src/dns_cache_handler.erl

```diff
@@ -0,0 +1,176 @@
+%%%-------------------------------------------------------------------
+%%% @author dmitry.pravosudov
+%%% @copyright (C) 2019, <COMPANY>
```

Comment:
Should <COMPANY> be something different, here?

Reply:
@ditas make sure the header file is similar to the other ones not need to add this.